index.bs

```diff
@@ -269,6 +269,45 @@ See also:
 
 * [[LEAST-POWER]]
 
+<h3 id="high-level-low-level">Resolving tension between high level and low level APIs</h3>
+
+<!--
+See "High level or low-level APIs?"
+https://github.com/w3ctag/design-principles/issues/117
+-->
+
+High-level APIs allow user agents more ability to
+intervene in various ways [on behalf of the user](#priority-of-constituencies),
+such as to ensure accessibility, privacy, or usability.
+
+<div class=example id=font-enumeration>
+
+A font picker (high level API)
+[was recommended by the TAG](https://github.com/w3ctag/design-reviews/issues/399)
+over a Font Enumeration API (low level API)
+as it addresses the bulk of use cases,
+without the fingerprinting concerns that accompany
+a general Font Enumeration API.
+
+</div>
+
+Low-level APIs afford authors room for experimentation
+so that what the high level APIs can organically emerge
+from usage patterns over time.
+They also provide an escape hatch when the higher-level API
+is not adequate for the use case at hand.
+
+Whenever possible, design high level APIs in terms of
+building blocks over lower level APIs.
```

Comment:
I think maybe this sentence should be moved after the next one? So that the reader knows *why* this is the case?

Reply:
Just pushed a change (somewhat larger than what you were suggesting), thoughts?